backend/src/main/java/elbin_bank/issue_tracker/issue/application/query/GetFilteredIssueListService.java

```diff
@@ -0,0 +1,71 @@
+package elbin_bank.issue_tracker.issue.application.query;
+
+import elbin_bank.issue_tracker.issue.application.query.dto.IssueSummaryDto;
+import elbin_bank.issue_tracker.issue.application.query.dto.IssuesResponseDto;
+import elbin_bank.issue_tracker.issue.domain.Issue;
+import elbin_bank.issue_tracker.issue.domain.IssueRepository;
+import elbin_bank.issue_tracker.issue.domain.IssueStatus;
+import elbin_bank.issue_tracker.label.application.query.dto.LabelsDto;
+import elbin_bank.issue_tracker.label.domain.Label;
+import elbin_bank.issue_tracker.milestone.domain.MilestoneRepository;
+import elbin_bank.issue_tracker.user.domain.UserRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+@Service
+public class GetFilteredIssueListService {
+
+    private final IssueRepository issueRepository;
+    private final UserRepository userRepository;
+    private final MilestoneRepository milestoneRepository;
+
+    public GetFilteredIssueListService(IssueRepository issueRepository, UserRepository userRepository, MilestoneRepository milestoneRepository) {
+        this.issueRepository = issueRepository;
+        this.userRepository = userRepository;
+        this.milestoneRepository = milestoneRepository;
+    }
+
+    @Transactional(readOnly = true)
+    public IssuesResponseDto find(String q) {
+        FilterCriteria crit = FilterConditionParser.parse(q);
+        List<Issue> issues = issueRepository.findByFilter(crit.isClosed());
+        if (issues.isEmpty()) {
+            return new IssuesResponseDto(List.of(), 0L, 0L);
+        }
+
+        List<Long> issueIds = issues.stream().map(Issue::getId).toList();
```

Comment:
nit) Java 21부터는 List나 Set 같은 Collection 타입에서 stream()을 굳이 호출하지 않아도 map, filter, toList() 같은 Stream-like 연산을 바로 사용할 수 있는 Sequenced Collection API 확장이 도입되었습니다.
```suggestion
        List<Long> issueIds = issues.map(Issue::getId).toList();
```

Reply:
직접 해본 결과 `Cannot resolve method 'map' in 'List'` List인터페이스 안에 map이 없다고 나오고 있습니다. 그래서 검색을 해보았는데 Java21에는 없고 코틀린에서 지원되는 기능이라고 합니다...!!

Reply:
흐음.. 어렴풋한 기억으로 코멘트했는데 제가 코틀린과 헷갈렸나봅니다 ㅎㅎ 감사합니다~!